aiida_quantumespresso/parsers/parse_xml/pw/parse.py

```diff
@@ -101,7 +101,7 @@ def parse_pw_xml_post_6_2(xml):
             logs.error.append(str(err))
 
     xml_version = StrictVersion(xml_dictionary['general_info']['xml_format']['@VERSION'])
-    inputs = xml_dictionary['input']
+    inputs = xml_dictionary.get('input',{})
```

Comment:
Can we check with QE (Pietro Delugas?) if these changes are intentional?

Reply:
The xml files that I have do not have the input section